src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -225,6 +227,43 @@ public async Task<ActionResult> ManageContactDetails(AatfEditContactAddressViewM
             return View(viewModel);
         }
 
+        [HttpGet]
+        public async Task<ActionResult> Delete(Guid id, Guid organisationId, FacilityType facilityType)
```

Comment:
Can we add the  [AuthorizeInternalClaims(Claims.InternalAdmin)] attribute to the action and create unit test

Maybe also add to the ManageContact details and ManageAatf details actions

Reply:
Added